### DISCUSSION TITLE
EAO=0.414

### DISCUSSION THREAD

**Discussion by ghost** [STVIR/pysot#94] [Reproduce]
how do you get the  EAO=0.414?
after load the resnet50.model and then train the coco vid det youtube_bb ?
which is the checkpointe_ ?

**Comment by mengmeng18** [comment]
i have the same question

**Comment by mengmeng18** [comment]
I only have EAO=0.328 in vot2018 in epoch20

**Comment by zhanght021** [comment]
@mengmeng18   the best always is not epoch 20

**Comment by mengmeng18** [comment]
thanks!

**Comment by zhanght021** [comment]
@mengmeng18  What is your best EAO in VOT2018？

**Comment by mengmeng18** [comment]
I just test on epoch 20, i will test it on other epoch later.What is your best EAO?

**Comment by zhanght021** [comment]
@mengmeng18  0.35 please let me know your best EAO when you test it tks

**Comment by ghost** [comment]
EAO=0.414 is base on 16GPU ? what about use siamrpn_r50_l234_dwxcorr_8gpu with 3 GPU?

**Comment by XiaoCode-er** [comment]
i trained the siamrpn_r50_l234_dwxcorr_8gpu  on vid, det, coco and got10k, got my best EAO=0.378 in epoch 14.

**Comment by mengmeng18** [comment]
I will let you know@zhanght021

**Comment by mengmeng18** [comment]
@XiaoCode-er,can you share how to train on got10k

**Comment by XiaoCode-er** [comment]
@mengmeng18 First, you need refer vid data processing to generate the crop511 and trian.json. Second, add related paramters of got10 to the config.py and config.yaml.

**Comment by zhanght021** [comment]
@XiaoCode-er , i add LASOT and GOT10K datasets to train ,but the EAO is only 0.35, do you do any other finetune?

**Comment by mengmeng18** [comment]
@XiaoCode-er,thanks a lot!

**Comment by XiaoCode-er** [comment]
i didn't do any finetune, maybe you can adjust the batch size, beacuse it is 128 in the paper but in config script is 32. @zhanght021

**Comment by XiaoCode-er** [comment]
@mengmeng18 ,
sorry, data and data process script were accomplished during my internship,
so, i can not send these to you due to company policy,
if you need know some detail, you can ask me at any time.

**Comment by mengmeng18** [comment]
@XiaoCode-er ,thanks for help!

**Comment by xiaotian3** [comment]
> i trained the siamrpn_r50_l234_dwxcorr_8gpu on vid, det, coco and got10k, got my best EAO=0.378 in epoch 14.

the same e14

**Comment by mengmeng18** [comment]
@XiaoCode-er,when crop Lasot, did you meet the problem as follows:
par_cropnew.py:45: RuntimeWarning: invalid value encountered in double_scalars
  c = -a * bbox[0]
par_cropnew.py:46: RuntimeWarning: invalid value encountered in double_scalars
  d = -b * bbox[1]

**Reply by sharon3-2** [reply]
hi, i wonder if training on the Got10k help the results? and have you adjusted hyper-parameters?

**Comment by XiaoCode-er** [comment]
@mengmeng18  i only processed got10k. 
@zhanght021 did you meet the crop lasot problem which was mentioned ?

**Comment by zhanght021** [comment]
@XiaoCode-er  the target is occluded ， and the width and height is 0，

**Comment by xi-mao** [comment]
> > i trained the siamrpn_r50_l234_dwxcorr_8gpu on vid, det, coco and got10k, got my best EAO=0.378 in epoch 14.
> 
> the same e14
 我想问下 为什么我训到第11epoch的时候程序员会自动停掉？

**Comment by zhanght021** [comment]
> > > i trained the siamrpn_r50_l234_dwxcorr_8gpu on vid, det, coco and got10k, got my best EAO=0.378 in epoch 14.
> > 
> > 
> > the same e14
> > 我想问下 为什么我训到第11epoch的时候程序员会自动停掉？

cuda out of memory??? if thirs problem , you should reduce the batch_size

**Comment by xi-mao** [comment]
@zhanght021 thanks, so what do you set your batchsize?

**Comment by ZhiyuanChen** [comment]
由于本issue没有得到更进一步的提问，我现在将其关闭。
如有新的问题，请重新打开它或者创建一个新的issue。

**Comment by ucasqcz** [comment]
i have trained on youtube, vid, coco, det with 8gpu, under config file  siamrpn_alex_dwxcorr_16gpu/config.yml, but only get 0.30 eao on VOT2018  without hp_search, do you have any suggestion?

**Comment by xi-mao** [comment]
> i have trained on youtube, vid, coco, det with 8gpu, under config file siamrpn_alex_dwxcorr_16gpu/config.yml, but only get 0.30 eao on VOT2018 without hp_search, do you have any suggestion?

I got the same results 0.30eao without  hp_search, train with tybb, vid, coco, det on two GPU.
I have train several times, the results is still 0.30 eao On vot2018

**Comment by zzzmm1** [comment]
I use the pretrained model siamrpn_r50_l234_dwxcorr, but only get 0.373 EAO and 57 lost number on VOT2018 under windows10 system, whats the probable problem？

**Comment by ZhiyuanChen** [comment]
It is hard to reproduce the result, but not impossible.

**Comment by ortegahz** [comment]
any updates?
